jscomp/syntax/tests/printer/expr/expected/dict.res.txt

```diff
@@ -17,14 +17,10 @@ let x = dict{
   "baz": baz,
 }
 
-let x = dict{"foo": "bar", "bar": "baz"}
-let x = dict{"foo": "bar", "bar": "baz", "baz": baz}
+let x = Js.Dict.fromArray([("foo", "bar"), ("bar", "baz")])
```

Comment:
not exactly what you want...

Reply:
Oh actually, in the printer that's expected since there's nothing anymore that converts between fromArray/dict literal. So maybe that's fine.

In the roundtrip tests though that should not happen.

Reply:
Seems fine to me that Js.Dict.fromArray is not "auto-converted" to dict{} anymore.

Reply:
Yeah this sounds correct.